tools/ci/run_tc.py

```diff
@@ -45,8 +45,7 @@
 import tempfile
 import zipfile
 
-sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
-from wpt.utils import get_download_to_descriptor
+from ..wpt.utils import get_download_to_descriptor
```

Comment:
OK this caused everything to fail with "ImportError: attempted relative import with no known parent package"

mypy can't figure out what `get_download_to_descriptor` is without this change, but I guess I'll go with `type: ignore` instead.